lib/inherited_resources.rb

```diff
@@ -21,18 +21,12 @@ module InheritedResources
   def self.flash_keys=(array)
     Responders::FlashResponder.flash_keys = array
   end
-
-  class Railtie < ::Rails::Engine
-    config.inherited_resources = InheritedResources
-    if config.respond_to?(:app_generators)
-      config.app_generators.scaffold_controller = :inherited_resources_controller
-    else
-      config.generators.scaffold_controller = :inherited_resources_controller
-    end
-  end
 end
 
-class ActionController::Base
+ActiveSupport.on_load(:action_controller) do
```

Comment:
We could move this to the engine and I think it would be better but I'm not sure if people uses inherit_resources without Rails. Do you think it is fine to us to move it there?